planner/core/plan_cost_ver2.go

```diff
@@ -329,6 +329,11 @@ func (p *PhysicalIndexMergeReader) getPlanCostVer2(taskType property.TaskType, o
 	sumIndexSideCost := sumCostVer2(indexSideCost...)
 
 	p.planCostVer2 = sumCostVer2(tableSideCost, sumIndexSideCost)
+	// give a bias to pushDown limit, since it will get the same cost with NON_PUSH_DOWN_LIMIT case via expect count.
+	// push down limit case may reduce cop request consumption if any in some cases.
+	if p.PushedLimit != nil {
+		p.planCostVer2 = mulCostVer2(p.planCostVer2, 0.99)
```

Comment:
We'd better implement this rule in physical optimization like `pushLimitOrTopNForcibly` instead of in the cost model. However the current `IndexMerge` implementation prevents us from doing this. Can we add a comment here to notify us to remove this tricky cost formula once we can implement this rule in the physical optimization?

Reply:
make sense, definitely